### PR TITLE
Fine Tuned Login UI and added Navbar and Leaderboard

### LINKED ISSUE #63: Finished Login UI
**Describe the solution you'd like**
Issue for current PR #62 

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by blaiadam**
**Description**
This is a rough draft of the final look for the Login page. I will move onto the other pages to ensure they have similar look and feel to this page. 

Two components are also added: Navbar and the Leaderboard.
The Navbar CSS is in main as that should remain the same throughout the pages it is used in. However, the component itself will need changes for the text that will be shown on it. If you are implementing this component, copy its template from the /src/Login folder. The current Navbar in the Login page is a template and displays information that won't be there in the final product.
The Leaderboard CSS will be different between pages, so that is in the local Login.css file. The component can also be copied from the /src/Login folder.

Another big change was moving Login specific CSS to the local Login.css file. Main.css was getting heavy and had unnecessary clutter in there. I also added into the Technical Document the structure of our CSS files to clarify to new users how we organized it. It is on page 26 if you are curious.
Lastly, I added a close button to the popups and messed around with the colors on the popups.

Fixes #63

**Type of change**
Check options that are relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Fix or feature that would cause existing functionality to not work as expected
- [x] This change requires a update in the design document

**Checklist:**
Go down the list and check them off.

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the design document (if applicable)
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: fortuna/src/login/Leaderboard.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+//@flow strict
+
+import * as React from 'react';
+import '../Main.css';
+import './Login.css';
+import 'bootstrap/dist/css/bootstrap.min.css';
+
+// Leaderboard component. Displays top 10 players.
+class Leaderboard extends React.Component<{||}> {
```
Comment (SecondThread) [inline fortuna/src/login/Leaderboard.js]: Solid, thanks for making this its own component

File: fortuna/src/Main.css [hunk 2]
```diff
@@ -7,34 +7,39 @@ body, html {
     background: #000921;
 }
 
-h1 {
-    padding: 5px 5px 5px 5px;
+/* Navbar Styling */
```
Comment (SecondThread) [inline fortuna/src/Main.css]: Stuff specific to the navbar should be in its own file imo. This file should be for generic stuff that anyone might want to use when then need a button, div et cetera.

File: fortuna/src/login/Navbar.js [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+// The middle column is the current page name.
+// The right column is the player's name and current currency.
+//
+// The columns must be customized for each page this component is used for.
+class Navbar extends React.Component<{||}> {
+	render(): React.Node {
+		return (
+			<div className="topnav">
+				<div className="navleft">
+					<button type="button" className="navbtn">&#60;&#45; Leave</button>
```
Comment (SecondThread) [inline fortuna/src/login/Navbar.js]: This would be a good thing to have be passed in as part of the Props. Thanks for making its own component though

**Review by SecondThread — APPROVED** [review]

**Review by Airor4 — APPROVED** [review]
Everything looks good although I'm not too sure about the frontend code but if David approved it thats good enough for me. Good job on the components!

**Comment by SleepyLime** [comment]
@blaiadam
Just as a reminder, to fix merge conflicts, move everything in the `src/` folder up from the `fortuna/` folder into the `Fortuna/` folder, then delete the old `fortuna/` folder

**Comment by Derplime** [comment]
Attempt 2:
@blaiadam
Just as a reminder, to fix merge conflicts, move everything in the `fortuna/` folder up into the `Fortuna/` folder, then delete the old `fortuna/` folder

File: fortuna/src/Main.css (hunk 2 above)
Comment (blaiadam) [reply fortuna/src/Main.css]: Gonna fix this issue in my next pull.

**Pull request merged**